src/pages/LocalDetail.tsx

```diff
@@ -0,0 +1,265 @@
+import styled from '@emotion/styled';
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
+import dfs_xy_conv from '@/components/Translate';
+import { DustState } from '@/components/Dust';
+
+import { type LocalDustDetail } from '@/type';
+
+const { VITE_WEATHER_API_KEY, VITE_API_KEY, VITE_KAKAO_API_KEY } = import.meta
+  .env;
+
+type Location = {
+  latitude: number;
+  longitude: number;
+};
+
+const LocalDetail = () => {
+  const [temperature, setTemperature] = useState();
+  const [humidity, setHumidity] = useState();
```

Comment:
타입을 명시해주시면 좋을 것 같아요. number라면

```suggestion
  const [temperature, setTemperature] = useState<number | null>(null);
  const [humidity, setHumidity] = useState<number | null>(null);
```